### PR TITLE
Generate an environment.json for OpenStack Heat stacks

### PR DESCRIPTION
**Pull request by kiall**
We'll end up with a slightly expanded environment.json file, to account for SSH bastions and public vs private IPs:

      {
         "dashboardHost" : "172.24.0.5",
         "sshUser" : "root",
         "sshKey" : "/home/kiall/SUSE/caasp/automation/openstack-heat/tools/id_docker",
         "kubernetesHost" : "10.84.72.173",
         "minions" : [
            {
               "index" : "0",
               "role" : "admin",
               "addresses" : {
                  "publicIpv4" : "10.84.72.172",
                  "privateIpv4" : "172.24.0.5"
               },
               "fqdn" : "test2-admin-xpi3oesblxxc.k8s.local",
               "minionId" : "67af839105f24307bb2bf55d603c55f0"
            },
            {
               "minionId" : "ac93ab2058914fad88d87702f8efc863",
               "addresses" : {
                  "publicIpv4" : "10.84.72.173",
                  "privateIpv4" : "172.24.0.6"
               },
               "fqdn" : "test2-master-7lydnr7r4sum.k8s.local",
               "role" : "master",
               "index" : "1"
            },
            {
               "minionId" : "618fb2611cfe41239be7cbdab853793c",
               "index" : "2",
               "role" : "worker",
               "proxyCommand" : "ssh root@10.84.72.172 -W %h:%p",
               "fqdn" : "test2-worker-eorkpgetfvr6-oxup2sra6zrr-azgmmxgbnbz4.k8s.local",
               "addresses" : {
                  "privateIpv4" : "172.24.0.7"
               }
            },
            {
               "minionId" : "3746ed898269450d8966543734da6554",
               "proxyCommand" : "ssh root@10.84.72.172 -W %h:%p",
               "addresses" : {
                  "privateIpv4" : "172.24.0.8"
               },
               "fqdn" : "test2-worker-eorkpgetfvr6-4rzhx3nww5fv-glz5hrejhktm.k8s.local",
               "role" : "worker",
               "index" : "3"
            }
         ]
      }

### CODE REVIEW

File: velum-bootstrap/README.md [hunk 1]
```diff
@@ -20,65 +20,15 @@ To make sure the tests know where to run against you need to specify a `environm
 ### 1. local cluster generated via [caasp-devenv](https://github.com/kubic-project/caasp-devenv) and [terraform](https://github.com/kubic-project/terraform)
 
 **NOTE**: for this setup make sure to remove provide `DEVENV=true` as all `docker` commands will run on localhost without any ssh access. You can just use the `environment.json` file generated by the [terraform](https://github.com/kubic-project/terraform) setup
-```json
```
Comment (kiall) [inline velum-bootstrap/README.md]: I removed these examples as they get out of date pretty quickly - there is a single example in the root of the repo.

File: caasp-devenv [hunk 2]
```diff
@@ -112,7 +112,7 @@ setup() {
   fi
 
   # || : is necessary, as Zypper exits non-zero for "no changes".
-  sudo zypper in --no-confirm python-tox ruby2.1-rubygem-bundler ruby2.1-devel phantomjs libxml2-devel libxslt-devel terraform terraform-provider-libvirt || :
+  sudo zypper in --no-confirm python-tox python-openstackclient ruby2.1-rubygem-bundler ruby2.1-devel phantomjs libxml2-devel libxslt-devel terraform terraform-provider-libvirt || :
```
Comment (grahamhayes) [inline caasp-devenv]: also need `python-heatclient`

Comment (kiall) [inline caasp-devenv]: Done

File: caasp-openstack-heat/caasp-openstack [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      ;;
+    -h|--help)
+      echo "$USAGE"
+      exit 0
+      ;;
+  esac
+  shift
+done
+
+[ -z $OPENRC_FILE ]  && error "Option --openrc is required"
```
Comment (grahamhayes) [inline caasp-openstack-heat/caasp-openstack]: Should we allow people to pre source openrc files?

Comment (kiall) [inline caasp-openstack-heat/caasp-openstack]: Maybe, but we can add that later if people want it :)

**Pull request merged**